package.json

```diff
@@ -33,13 +33,16 @@
     "@babel/plugin-syntax-dynamic-import": "^7.2.0",
     "@babel/preset-env": "^7.2.3",
     "babel-eslint": "^10.0.1",
+    "babel-jest": "^24.1.0",
     "babel-loader": "^8.0.4",
+    "babel-plugin-dynamic-import-node": "^2.2.0",
     "clean-webpack-plugin": "^1.0.0",
     "custom-event": "^1.0.1",
     "eslint": "5.11.1",
     "eslint-config-canopy": "2.1.0",
     "in-publish": "^2.0.0",
     "jasmine": "^3.3.1",
+    "jest": "^24.1.0",
     "karma": "^3.1.4",
```

Comment:
we should remove this dependency and all of the other karma dependencies. Maybe even the `promise-polyfill` dependency